src/js/components/Products/Product.scss

```diff
@@ -0,0 +1,61 @@
+$primary-color: #fff;
+
+.Product{
+    background-color: #bbb;
+    min-height: 100vh;
+    color: $primary-color;
+}
+span {
+  display:block;
+  overflow: hidden;
+  padding: 0 4px 0 6px;
+  .rad {
+    display: flex;
+    flex-grow: 20;
+  }
+}
+.checkboxs{
+  display: flex;
+}
+.color{
+  display: flex;
+}
+.createButton{
+    float: right;
+  }
+.rad{
+  display: flex;
+  flex-grow: 10;  
+}
+label {
```

Comment:
Styling elements mean it will be applied to all elements of this type. This could make future development difficult rather make use of classes and or identifiers instead of styling an element directly

Reply:
I have changed the element to classsname for styling.